src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -205,9 +205,12 @@ private static void mapTransitPreferences(NodeAdapter c, TransitPreferences.Buil
               .summary("The extra time after exiting a public transport vehicle.")
               .description(
                 """
-The slack is added to the time when going from the transit vehicle to the stop.
-This also influences the total time it takes to transfer. See `transferSlack` for more details about
-how the total transfer time is calculated.
+The slack is added to the time after leaving the transit vehicle.
+
+This also influences the time it takes to transfer.
+
+Since some modes, like airplanes and subways, need more time than others, this is also configurable
+per mode with `boardSlackForMode`.
```

Comment:
alightSlackForMode*, for some reason there isn't suggestion option when adding review comment :thinking: .

Reply:
Ah I think it was because I was reviewing commit instead of overall changes.